packages/dds/tree/src/test/rebaserAxiomaticTests.ts

```diff
@@ -524,11 +523,12 @@ function sandwichRebaseWithCompose<TChangeset>(
 			...sourceEdits.slice(0, i),
 		];
 		const rebaseMetadata = rebaseRevisionMetadataFromInfo(
-			defaultRevInfosFromChanges(rebasePath),
+			defaultRevInfosFromChanges([...rebasePath, edit]),
```

Comment:
Adding the info from `edit` here doesn't match the doc comment for that argument on rebaseRevisionMetadataFromInfo. Is the doc comment not correct?

Reply:
This PR changes the contract. I've updated the comment.